src/Interfaces/ISDK.ts

```diff
@@ -16,4 +16,5 @@ export default interface ISDK {
   emailTranscript(requestId: string, token: string, emailRequestBody: object): Promise<void>;
   fetchDataMaskingInfo(requestId: string): Promise<IDataMaskingInfo>;
   makeSecondaryChannelEventRequest(requestId: string, secondaryChannelEventRequestBody: object): Promise<void>;
+  sendTypingIndicator(requestid: string): Promise<void>;
```

Comment:
nit: `requestId`  instead of `requestid`

Reply:
updated